src/manage_sql_nvts.c

```diff
@@ -1496,6 +1497,65 @@ manage_update_nvt_cache_osp (const gchar *update_socket)
                " VALUES ('checked_preferences', 1)"
                " ON CONFLICT (name) DO UPDATE SET value = EXCLUDED.value;");
         }
+
```

Comment:
I think this should come before telling the main process to update its NVT cache.

Reply:
Okay, I've changed it in 188e38bf9ea95129cc036d5c185823e9b25f8c8d.